File: Harvard_CS50_Intro/Week 4/Lecture 4/compare1.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    printf("s: ");
    char *s = get_string();
 
    printf("t: ");
    char *t = get_string();
 
    if (s != NULL && t != NULL)
    {
        // compare two char * sequence
        if (strcmp(s, t) == 0)
        {
            printf("same\n");
        }
        else
        {
            printf("different\n");
        }
    }
}

File: Harvard_CS50_Intro/Week 8/Lecture 8/week4/compare1.py
import cs50

print("s: ", end="")
s = cs50.get_string()

print("t: ", end="")
t = cs50.get_string()

# Python does not have null, it uses None

if s != None and t != None:
    if (s == t):
        print("same")
    else:
        print("different")
